proctor-common/src/main/java/com/indeed/proctor/common/ForceGroupsOptionsStrings.java

```diff
@@ -80,8 +232,46 @@ public static String generateForceGroupsString(final ForceGroupsOptions options)
         options.getDefaultMode().getToken()
                 .ifPresent(tokens::add);
 
+        final Map<String, Payload> forcePayloads = options.getForcePayloads();
+
         options.getForceGroups()
-                .forEach((testName, bucketValue) -> tokens.add(testName + bucketValue));
+                .forEach((testName, bucketValue) ->
+                        tokens.add( forcePayloads.containsKey(testName) ? testName + bucketValue + ";" +
+                                createForcePayloadString(forcePayloads.get(testName)) : testName + bucketValue));
         return String.join(",", tokens);
     }
+
+    @Nonnull
+    private static String createForcePayloadString(final Payload payload) {
+        final StringBuilder s = new StringBuilder(50);
```

Comment:
As similar as https://github.com/indeedeng/proctor/pull/76#discussion_r948126024,

Can this method be a bit easier to understand if we use ObjectMapper?
```
if (payload.getStringArray() != null) {
    s.append("stringArray:")
            .append(mapper.writeValueAsString(payload.getStringArray()));
}
```

Reply:
I made this change to String and String array to take into account how `mapper.readValue()/writeValue()` handles strings. This ensures that the generated and read force group strings are equal. Should I also apply this to double, double[], long, and long[] for formatting?